dev/integration_tests/wide_gamut_test/README.md

```diff
@@ -0,0 +1,10 @@
+# wide_gamut_test
+
+An integration test used for testing wide gamut color support in the engine.
+
+## Local run
+
+```sh
+flutter create --platforms="ios" --no-overwrite .
```

Comment:
Does this work? I thought flutter create errors in the Flutter repo.

Reply:
Works for me and the devicelab test.  *shrug*